packages/x-codemod/README.md

```diff
@@ -90,6 +90,7 @@ The list includes these transformers
 - [`view-components-rename-value-prop`](#view-components-rename)
 - [`localization-provider-rename-locale`](#localization-provider-rename-locale)
 - [`text-props-to-localeText`](#text-props-to-localeText)
+- [`replace-tabs-props](#replace-tabs-props)
```

Comment:
This codemod is not in `rename-components-to-slots` because it's not a breaking change.

The idea would be to encourage doing it at the end, and include it in the v7 release.

Reply:
Hm. It is indeed not a breaking change, but at the same time—a deprecated feature, that can easily be resolved.
I feel that if we make it optional and only mention it in the migration doc—we'd end up with half people using slots and other half sticking with components. 🤔 
But sure, we can try going this route with only suggesting this migration as an optional step. 👌